CIScripts/Tests/CommonTestCode.ps1

```diff
@@ -11,6 +11,10 @@ class NetAdapterInformation : NetAdapterMacAddresses {
 class ContainerNetAdapterInformation : NetAdapterInformation {
     [string] $AdapterShortName;
     [string] $AdapterFullName;
+    [string] $IfName;
```

Comment:
`ContainerNetAdapterInformation` already has all this fields. It is derived from `NetAdapterInformation` class, which is derived from `NetAdapterMacAddresses`.